src/flagStore.ts

```diff
@@ -146,10 +165,11 @@ export class FlagStore {
 	}
 
 	private ldConfig(): Record<string, number | string | boolean | LaunchDarkly.LDFeatureStore> {
+		const streamUri = this.config.baseUri.replace('app', 'stream');
```

Comment:
Are you recommending we remove the stream config option? I'm not sure this replace always be accurate in a development environment.